src/org/thoughtcrime/securesms/components/ThumbnailView.java

```diff
@@ -257,6 +257,12 @@ protected void onPostExecute(Boolean success) {
           }
         }.executeOnExecutor(AsyncTask.THREAD_POOL_EXECUTOR);
       }
+      if(slide.hasSticker())
```

Comment:
it turned out that the lottie stickers were not animated because the transition (`.transition(withCrossFade())`) I just removed the transition for stickers, if someone knows more about glide and have an idea about how to make the lottie drawable work with the transitions, patches are welcome :D